src/core/components/hv-navigator/index.tsx

```diff
@@ -174,20 +174,20 @@ export default class HvNavigator extends PureComponent<Types.Props> {
       | TypesLegacy.DOMString
       | null
       | undefined = props.element.getAttribute('type');
-    const initial:
+    const selected:
       | TypesLegacy.Element
-      | undefined = NavigatorService.getInitialNavRouteElement(props.element);
-    if (!initial) {
+      | undefined = NavigatorService.getSelectedNavRouteElement(props.element);
+    if (!selected) {
       throw new NavigatorService.HvNavigatorError(
-        `No initial route defined for '${id}'`,
+        `No selected route defined for '${id}'`,
```

Comment:
Is the `selected` <-> `initial` change intentional?

Reply:
Yup!